lua/lazy-lsp/helpers.lua

```diff
@@ -168,6 +236,8 @@ end
 return {
   server_configs = server_configs,
   in_shell = in_shell,
+  in_shell_extendable = in_shell_extendable,
+  nix_command_available = nix_command_available,
```

Comment:
It does not seem like `nix_command_available` should be a part of public API, so we should not expose it.